kms/encrypt_symmetric.go

```diff
@@ -40,21 +41,41 @@ func encryptSymmetric(w io.Writer, name string, message string) error {
 	// Convert the message into bytes. Cryptographic plaintexts and
 	// ciphertexts are always byte arrays.
 	plaintext := []byte(message)
+	
+	// Optional, but recommended: Compute plaintext's CRC32C.
+	// See crc32c() function below.
+	plaintextCrc32c = crc32c(plaintext)
```

Comment:
Might be better to inline the `crc32c` function to keep the sample as simple as possible.

Reply:
Chatted offline - since the function is used twice in the snippet, we'll keep it as a standalone function.